backend/measurement-registry/src/main/resources/measurement-persister.yml

```
@@ -0,0 +1,482 @@
+            - jsonPath: $.measurements.*.documents.*.documentUid
+
+            - jsonPath: $.measurements.*.documents.*.additionalDetails
+              type: JSON
+              dbType: JSONB
+
+            - jsonPath: $.measurements[*][?({id} in @.documents[*].id)].auditDetails.lastModifiedTime
+
+            - jsonPath: $.measurements[*][?({id} in @.documents[*].id)].auditDetails.lastModifiedBy
+
+
+
```

YAMLlint (1.37.1)

[error] too many blank lines (3 > 2)

(empty-lines)